sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json

```diff
@@ -192,7 +192,7 @@
             "get": {
                 "summary": "Retrieves statistics about device identities in the IoT hub’s identity registry.",
                 "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-                "operationId": "RegistryManager_GetDeviceStatistics",
+                "operationId": "IoTHubService_RegistryManager_GetDeviceStatistics",
```

Comment:
There is supposed to be a "statisitics" sub client here instead of "RegistryManager"